projectile.el

```diff
@@ -997,17 +1028,46 @@ function is executing."
          (list it))))
    (directory-files directory t)))
 
-(defun projectile-remove-ignored (files)
+(defun projectile-adjust-files (files)
+  "First remove ignored files from FILES, then add back unignored
+files."
+  (projectile-add-unignored (projectile-remove-ignored files)))
+
+(defun projectile-remove-ignored (files &optional subdirectories)
   "Remove ignored files and folders from FILES.
 
-Operates on filenames relative to the project root."
+Operates on filenames relative to the project root. Optionally,
```

Comment:
In Emacs you should use 2 spaces after a `.`. I know it's silly, but it's the standard...